packages/node-integration-tests/suites/express/tracing/server.ts

```diff
@@ -25,6 +25,10 @@ app.get(/\/test\/regex/, (_req, res) => {
   res.send({ response: 'response 2' });
 });
 
+app.get(['/test/array1', /\/test\/array[2-9]/], (_req, res) => {
```

Comment:
Can we test some more complicated regex? Perhaps with multiple segments?

Reply:
Sure thing, will add a few more tests